src/main/java/io/codelex/loops/practice/Piglet.java

```diff
@@ -0,0 +1,43 @@
+package io.codelex.loops.practice;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Piglet {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        Random r = new Random();
+
+        System.out.println("Welcome to Piglet!");
+        int score = 0;
+        boolean rollAgain = true;
+
+        while (rollAgain) {
+            int roll = r.nextInt(6) + 1;
+            System.out.println("You rolled a " + roll + "!");
+
+            if (roll == 1) {
+                System.out.println("You got 0 points.");
+                rollAgain = false;
+            } else {
+                score += roll;
+                System.out.println("Roll again? (yes/no)");
+
+                String nextRoll = "";
+                while (!nextRoll.equals("yes") && !nextRoll.equals("no") && !nextRoll.equals("no") && !nextRoll.equals("n")) {
```

Comment:
Varbūt to, kas ir while cikla iekavās iznes citā metodē, kura saņem nextRoll, kā parametru un atgriež boolean?
Jebšu kaut kā tā :
```
private static boolean isInputValid(String nextRoll) {
   return  !nextRoll.equals("yes") && !nextRoll.equals("no") && !nextRoll.equals("no") && !nextRoll.equals("n");
}
```
Tad nebūs 2x jāatkārto tas pats.